.landscaper/deploy_items.yaml

```diff
@@ -0,0 +1,62 @@
+- name: registration
+  type: Helm
+  config:
+    apiVersion: helm.deployer.landscaper.gardener.cloud/v1alpha1
+    kind: ProviderConfiguration
+    kubeconfig: {{ .imports.cluster.spec.config.kubeconfig | b64enc }}
+    repository: {{ index .cd.localResources "provider-aws" | .access.image }}
+    version: {{ index .cd.localResources "provider-aws" | .access.tag }}
+    name: gardener-extension-provider-aws
+    namespace: garden
+    values:
+      controllerRegistration:
+        deployment:
+          chart: {{ resolve(index .cd.localResources "provider-aws-chart" | .access.imageReference) }}
+          values:
+            image:
+              repository: {{ index .cd.localResources "provider-aws" | .access.image }}
```

Comment:
```suggestion
              repository: {{ index .cd.component.localResources "provider-aws" | .access.image }}
```

Reply:
@ccwienk currently the access for an oci hsa this format:
```
access:
  type: ociRegistry
  imageReference: myimage:1.0.0
```
With that format we do not have direct access to the repoository and the tag.
Should be add that to the component descriptor or should we provide a helper function to parse the tag and repository from the reference?

Reply:
@schrodit : As those image-URLs are standardised (from OCI), and widely-used (I think..), and reasonably easy to parse, I would keep it like this, if our "users" prefer it this way.